### PR TITLE
add singlecell flavor

### PR DESCRIPTION
**Pull request by bgruening**
A start. ping @bebatut and @mtekman 

* [x] logo
* [ ] add more raceid tools, stemID, fateID?
* [x] scanpy

### CODE REVIEW

File: index-singlecell.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+- [Plates, Batches, and Barcodes](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html){:target="_blank"}
+- [Understanding Barcodes](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-umis/tutorial.html){:target="_blank"}
+- [Pre-processing of Single-Cell RNA Data](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.html){:target="_blank"}
+
+
+# Contributors
+
+- [Bérénice Batut](https://github.com/bebatut)
+- [Mehmet Tekman](https://github.com/mtekman)
+- [Maria Doyle](https://github.com/mblue9)
```
Comment (bgruening) [inline index-singlecell.md]: @mblue9 are you ok with this?

File: index-singlecell.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+- [Pablo Moreno](https://github.com/pcm32)
```
Comment (bgruening) [inline index-singlecell.md]: @pcm32 are you ok with this and could you tell me the real name of NH3 :)

File: index-singlecell.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+- [NH3 :)](https://github.com/nh3)
+- [Christophe Antoniewski](https://github.com/drosofff)
```
Comment (bgruening) [inline index-singlecell.md]: @drosofff are you ok with that?

File: index-singlecell.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+- [Lea Bellenger](https://github.com/bellenger-l)
```
Comment (bgruening) [inline index-singlecell.md]: @bellenger-l are you ok with that?

File: index-singlecell.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+- [Jelle Scholtalbers](https://github.com/scholtalbers)
```
Comment (bgruening) [inline index-singlecell.md]: @scholtalbers ok you ok with that?

File: index-singlecell.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+- [Charles Girardot](https://github.com/cgirardot)
```
Comment (bgruening) [inline index-singlecell.md]: @cgirardot please confirm :)

File: index-singlecell.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+- [Nicola Soranzo](https://github.com/nsoranzo)
```
Comment (bgruening) [inline index-singlecell.md]: @nsoranzo ok? :)

File: index-singlecell.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+- [Graham Etherington](https://github.com/ethering)
```
Comment (bgruening) [inline index-singlecell.md]: @ethering are you ok that me mention you here?

File: index-singlecell.md (hunk 7 above)
Comment (nsoranzo) [reply index-singlecell.md]: Sure!

File: index-singlecell.md (hunk 6 above)
Comment (cgirardot) [reply index-singlecell.md]: sure thx !

**Comment by erasche** [comment]
https://singlecell.usegalaxy.eu/ is live, 404 will disappear upon merge.

File: index-singlecell.md (hunk 5 above)
Comment (scholtalbers) [reply index-singlecell.md]: Yes thanks! 👍

File: index-singlecell.md (hunk 8 above)
Comment (ethering) [reply index-singlecell.md]: Yes, absolutely. That's great.

File: index-singlecell.md (hunk 2 above)
Comment (pcm32) [reply index-singlecell.md]: NH3 is Ni Huang.

**Comment by pcm32** [comment]
I think that we should try to have another call soon to re-align efforts... so many months in radio silence probably have brought us a lot of redundancy in terms of development.

File: index-singlecell.md (hunk 2 above)
Comment (bgruening) [reply index-singlecell.md]: Thanks added him.

@nh3 is that ok for you as well?

**Comment by bebatut** [comment]
I added a possible logo there:

screenshot 2019-02-26 at 17 30 34

File: index-singlecell.md (hunk 3 above)
Comment (drosofff) [reply index-singlecell.md]: Yes 👍 . Thanks !

**Comment by bgruening** [comment]
@pcm32;

> I think that we should try to have another call soon to re-align efforts... 

Good idea, doodle to find a date?

> so many months in radio silence probably have brought us a lot of redundancy in terms of development.

I hope not! :)

File: index-singlecell.md (hunk 1 above)
Comment (mblue9) [reply index-singlecell.md]: Yes thanks!

File: index-singlecell.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+
+# Contributors
+
+- [Bérénice Batut](https://github.com/bebatut)
+- [Mehmet Tekman](https://github.com/mtekman)
+- [Maria Doyle](https://github.com/mblue9)
+- [Pablo Moreno](https://github.com/pcm32)
+- [Ni Huang](https://github.com/nh3)
+- [Jonathan Manning](https://github.com/pinin4fjords)
```
Comment (bgruening) [inline index-singlecell.md]: @pinin4fjords is that ok for you?

File: index-singlecell.md (hunk 4 above)
Comment (bellenger-l) [reply index-singlecell.md]: Yes, thanks !

File: index-singlecell.md (hunk 9 above)
Comment (pinin4fjords) [reply index-singlecell.md]: Yep, thank you!

File: index-singlecell.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+
+# Welcome to the world of Single Cell Omics
+{:.no_toc}
+
+The Sincle Cell Omics workbench is a comprehensive set of analysis tools and consolidated workflows. The workbench is based on the [Galaxy framework](https://galaxyproject.org/), 
+which guarantees simple access, easy extension, flexible adaption to personal and security needs, and sophisticated analyses independent of command-line knowledge.
+
+The current implementation comprises more than 20 bioinformatics tools dedicated to different research areas of single cell biology.
+
+This service is a joint project between different groups from the [Earlham Institute](http://www.earlham.ac.uk), the [EBI](https://www.ebi.ac.uk), [EMBL](https://gbcs.embl.de) the [Sorbonne University](http://drosophile.org/) and the [University of Freiburg](https://galaxyproject.eu/freiburg/).
```
Comment (drosofff) [inline index-singlecell.md]: > the [Sorbonne University](http://drosophile.org/)

the [Sorbonne University](http://artbio.fr/)

Comment (bgruening) [reply index-singlecell.md]: Changed that ...

**Comment by pcm32** [comment]
@bgruening could you please change [EBI] to [EMBL-EBI]. Thanks!

**Comment by bgruening** [comment]
Done.

**Comment by bgruening** [comment]
I'm going to merge this. We still have a few pending PRs for the training and including seurat would be nice too, but I guess this is a first nice version that we can all build on.

**Pull request merged**